2016/proposals/pravendra-singh.md

```diff
@@ -0,0 +1,220 @@
+# A survey responses visualizer application
+
+## Abstract
+
+Software Carpentry has been educating multidisciplinary researchers with the necessary computing skills to help them in their venture. It enables the same via workshops for different fields, community discussion between volunteers, and curated readings etc.
+
+The number of people attending the workshops itself is sufficient enough to help us imagine the impact Software Carpentry Foundation has been making since its foundation.
+
+![number-of-learners](http://software-carpentry.org/files/graphs/learners-2015-10.png)
+[credits: software-carpentry.org]
+
+Software Carpentry maintains a set of tools that help them manage their workshop events. For example, the project [AMY](https://github.com/numfocus/amy) is a workshop management tool for the organization.
+
+There are *pre* and *post* workshop surveys also. They support in the proper event organization by collecting details about expected attendees for *pre-workshop* surveys and feedback about the event for *post-workshop* surveys.
+
+The following proposal talks down an application which can "*Represent the survey responses in a comprehensible manner, assisting the workshop organizers (hosts) to plan the event according to the feedback from attendees*".
+
+The application will be a relieve from the current cumbersome method we use for data aggregation.
+
+It has an additional benefit here, as we (Software Carpentry) believe in openness, the resultant application can be open sourced for the great good of others who want to utilize the SurveyMonkey survey responses in an interactive fashion.
+
+## Technical Details
+
+The application will be powered by Django, as we also want it to work with AMY. It will use Plotly.js for visualizing different response types, which is a open source [project](https://plot.ly/javascript/open-source-announcement/).
+
+Going by the "*Object-oriented analysis and design*" approach for system design, we can categorize the application in the following sections.
+
+###### 1. Visual representation of the survey questions
```

Comment:
Ohh, I have switched it to Header-5 because the markdown rendered text color was looking nice.
An abject apology if it was wrong, I'll change it to just a simple ordered list.